Surface_mesh_segmentation/package_info/Surface_mesh_segmentation/dependencies

```diff
@@ -12,6 +12,7 @@ Kernel_23
 Modular_arithmetic
 Number_types
 Point_set_processing_3
+Principal_component_analysis_LGPL
```

Comment:
This line should not be in `Surface_mesh_segmentation` but in `Surface_mesh_skeletonization`.